ddtrace/opentracer/utils.py

```diff
@@ -1,4 +1,7 @@
-import ddtrace
+import ddtrace.contrib.asyncio
```

Comment:
```suggestion
# DEV: If `asyncio` or `gevent` are unavailable we do not throw an error,
#    `context_provider` will just not be set and we'll get an `AttributeError` instead
import ddtrace.contrib.asyncio
```

This was the only difference I had from what you have here.

Reply:
Do you want me to add this comment to my PR or do you prefer to close my PR in favor of yours?

Reply:
Your call on whether you think this comment is necessary. I am happy to accept either way.